src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -195,41 +197,23 @@ def wrapper(*args, **kwargs):
                 except (SystemErrorException, UserErrorException) as e:
                     # Throw inner wrapped exception directly
                     raise e
-                except (RateLimitError, ServiceUnavailableError, APIError, Timeout, APIConnectionError) as e:
+                except (RateLimitError, APITimeoutError) as e:
                     #  Handle retriable exception, please refer to
                     #  https://platform.openai.com/docs/guides/error-codes/api-errors
-                    #  Use default Timeout 600s, refer to
-                    #  https://github.com/openai/openai-python/blob/d1c36582e82cfa97568d7e9184454ee3b77975fc/openai/api_requestor.py#L37  # noqa
                     print(f"Exception occurs: {type(e).__name__}: {str(e)}", file=sys.stderr)
-                    if isinstance(e, RateLimitError) and getattr(e.error, "type", None) == "insufficient_quota":
```

Comment:
Please revisit all these changes and make sure no regression.

Reply:
revisit all